subprojects/passim.wrap

```diff
@@ -0,0 +1,4 @@
+[wrap-git]
+directory = passim
+url = https://github.com/hughsie/passim.git
+revision = main
```

Comment:
I think we should gate merging this in fwupd until it's at least a tagged version.

Reply:
100% yes.

Reply:
Fixed, as I did a 0.1.0 release just now.